doc/user_guide/checkers/features.rst

```diff
@@ -876,6 +876,9 @@ Refactoring checker Messages
   Emitted when accessing only the first or last element of str.split(). The
   first and last element can be accessed by using str.split(sep, maxsplit=1)[0]
   or str.rsplit(sep, maxsplit=1)[-1] instead.
+:consider-using-augmented-assign (C0210): *Use '%s' to do an augmented assign directly*
```

Comment:
Isn't it weird this change shows up in a patch release? Did we forget to update the docs last release?

Reply:
It's cherry-picked from main and should stay on main, let me fix that.